src/System.Private.CoreLib/shared/System/ValueTuple.cs

```diff
@@ -1447,7 +1448,7 @@ public bool Equals(ValueTuple<T1, T2, T3, T4, T5, T6> other)
                 && EqualityComparer<T6>.Default.Equals(Item6, other.Item6);
         }
 
-        bool IStructuralEquatable.Equals(object other, IEqualityComparer comparer)
+        bool IStructuralEquatable.Equals(object? other, IEqualityComparer comparer)
```

Comment:
Is this missing argument validation for comparer? If null it would nullref, no?

Reply:
Yes.  That happens in lots of places, some on purpose, some probably accidental / without thinking about it.  Unless/until it shows up as a real problem, we shouldn't change them.